src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$main-font: 'Roboto', 'Arial', sans-serif;
+$main-color-text: rgba(6, 11, 53, 1);
+$main-font-weigh: 400;
```

Comment:
There is a typo in the variable name `$main-font-weigh`. It should be `$main-font-weight` to follow standard naming conventions.